deploy/manifests/dev/us-east-2/tenant/storetheindex/dhstore-cluster/envoy/pod-monitor.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: envoy
+  labels:
+    app: envoy
+spec:
+  selector:
+    matchLabels:
+      app: envoy
+  namespaceSelector:
+    matchNames:
+      - storetheindex
+  podMetricsEndpoints:
+    - path: /metrics
```

Comment:
The path at which metrics are exposed in envoy is also different. This value should be the same as the one in `ServiceMonitor`

Reply:
I did copy this from [service-monitor.yaml](https://github.com/ipni/storetheindex/blob/main/deploy/manifests/dev/us-east-2/tenant/storetheindex/dhstore-cluster/instances/service-monitor.yaml), I think you mean I need to use what is in [envoy-monitor.yaml](https://github.com/ipni/storetheindex/blob/main/deploy/manifests/dev/us-east-2/tenant/storetheindex/dhstore-cluster/envoy/envoy-monitor.yaml)

Reply:
Done